.gitignore

```diff
@@ -4,6 +4,7 @@
 /node_modules
 /public/meta.json
 /src/consts/consts.js
+/src/consts/consts
```

Comment:
actually, we don't want this in the gitignore. there is a consts.js.example in that folder that does get checked in.